src/terms

```diff
@@ -0,0 +1,89 @@
+Data stored via the Services will be available via IPFS and stored on Filecoin until the expiry of the respective deals. There may be a delay between content being uploaded via the Services and being stored on Filecoin through a deal. Additionally, you may experience minor delays immediately after uploading new content to the Services while this content becomes available on the IPFS network.
+
+The maximum size of a registered upload is 750GB. Uploads greater than this size may not be registered successfully with your account, though if the data is successfully stored with web3.storage, you will still be responsible for paying for this storage.
+
+We reserve the right to terminate the Services at our sole discretion or to transfer operation of the Services to a third party or smart contract. Prior to termination or transfer, we will provide 90 days’ notice to users via email to allow users enough time to make arrangements if any (e.g. remote pinning to their own IPFS node, remote pinning to another service, or preparing to manage the renewal of their Filecoin deals on their own).
+
+All data uploaded to web3.storage is available to anyone who requests it using the correct CID. **Users should not store any private or sensitive information in an unencrypted form using web3.storage.** Deleting files from web3.storage via the site's web console or API will remove them from the file listing for a user's account, but you may be charged for at least 30 days of storage for those files. Additionally, uploaded files typically remain on our infrastructure for at minimum thirty (30) days, and nodes on the IPFS network may retain copies of the data indefinitely. **Users should not use web3.storage to store data that may need to be permanently deleted in the future.**
+
+#### **Paid Accounts**
+
+<u>Billing.  </u>
+ In order to begin storing files via the Services, you will need to provide us with one or more Payment Methods. "Payment Method" means a current, valid method of payment, which may involve a third party or payment processing vendor. You can increase your storage limit and add paid features to your account by signing up for a paid subscription plan (“Paid Plan”). 
+
+You agree that you’ll pay the amounts set forth in https://web3.storage/account/payment/ for your chosen tier (“Subscription Charge”) plus any charges for additional usage beyond the storage limit of your selected tier during the previous month (“Overages”) (collectively, “Subscription Charge” and “Overages” are referred to as (the “Fees”) for each billing cycle). 
+
+When determining tier storage limits and Overages, we track usage for payment using the volume of data associated with shard content identifiers/CIDs (CIDs referencing the actual data sent to web3.storage and stored) – usage is not tracked based on content CIDs (CIDs used to reference and access uploads in a format useful to users, like specific files and directories). See more detail on this point here(ADD LINK).
```

Comment:
ADD LINK

Reply:
@dchoi27 which reference link did you have in mind?

Reply:
ah good catch. it should be a link to a section of docs explaining store vs upload. we can remove for launch if too much to coordinate and put back after

Reply:
I am removing for now until we have docs for this, maybe a FAQ